src/number-input/number.component.spec.ts

```diff
@@ -129,6 +152,29 @@ describe("Number", () => {
 		expect(component.value).toEqual(0);
 	});
 
+	it("should decrement and set value to max if value - step is less than max", () => {
+		fixture.detectChanges();
+		buttonUp = fixture.debugElement.query(By.css(".up-icon")).nativeElement;
```

Comment:
Should this be `.down-icon`? 🤔

Reply:
Good catch, I'll update the test. Oversight from my side.

Reply:
fixed, could you review once more?